.github/workflows/ci.yml

```diff
@@ -28,16 +28,8 @@ jobs:
           cache: 'yarn'
 
       - name: Install dependencies
-        run: yarn
+        run: yarn install --prefer-offline --frozen-lockfile
```

Comment:
do you also want to add a `--link-duplicates` flag to hard link the repeated node modules? Other flags seem good

Reply:
I've heard of weird symlink issues when this is used, and since no one is using `--link-duplicates` on their local machine, i say let's run our ci in a similar fashion.

Reply:
Gonna resolve this thread, but pls reopen if you feel we should use this.